[2025-06-23T14:52:07.129+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-23T14:52:07.167+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_self_dag.task2 scheduled__2025-06-22T00:00:00+00:00 [queued]>
[2025-06-23T14:52:07.185+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_self_dag.task2 scheduled__2025-06-22T00:00:00+00:00 [queued]>
[2025-06-23T14:52:07.187+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-06-23T14:52:07.210+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): task2> on 2025-06-22 00:00:00+00:00
[2025-06-23T14:52:07.224+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=532) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-23T14:52:07.226+0000] {standard_task_runner.py:72} INFO - Started process 534 to run task
[2025-06-23T14:52:07.227+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_self_dag', 'task2', 'scheduled__2025-06-22T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag.py', '--cfg-path', '/tmp/tmpd5s70ala']
[2025-06-23T14:52:07.229+0000] {standard_task_runner.py:105} INFO - Job 5: Subtask task2
[2025-06-23T14:52:07.311+0000] {task_command.py:467} INFO - Running <TaskInstance: my_self_dag.task2 scheduled__2025-06-22T00:00:00+00:00 [running]> on host ef8c6abb536a
[2025-06-23T14:52:07.428+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Om' AIRFLOW_CTX_DAG_ID='my_self_dag' AIRFLOW_CTX_TASK_ID='task2' AIRFLOW_CTX_EXECUTION_DATE='2025-06-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-22T00:00:00+00:00'
[2025-06-23T14:52:07.429+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-23T14:52:07.461+0000] {logging_mixin.py:190} INFO - I am fine.
[2025-06-23T14:52:07.462+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-06-23T14:52:07.480+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-23T14:52:07.482+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=my_self_dag, task_id=task2, run_id=scheduled__2025-06-22T00:00:00+00:00, execution_date=20250622T000000, start_date=20250623T145207, end_date=20250623T145207
[2025-06-23T14:52:07.524+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-23T14:52:07.549+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-23T14:52:07.552+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
